config/adminlte.php

```diff
@@ -78,7 +79,7 @@
     | Here we change the layout of your admin panel.
     |
     | For more detailed instructions you can look here:
-    | https://github.com/jeroennoten/Laravel-AdminLTE/#65-layout
+    | https://github.com/jeroennoten/Laravel-AdminLTE/#71-layout
```

Comment:
https://github.com/jeroennoten/Laravel-AdminLTE/wiki/7.-Layout-and-Styling-Configuration